src/main/java/org/terasology/dynamicCities/region/RegionEntityManager.java

```diff
@@ -273,10 +272,8 @@ public void setNameTagForRegion(EntityRef region) {
         region.addComponent(nT);
     }
 
-    @Command(shortDescription = "Toggles the view of nametags for region entities", runOnServer = true,
-            requiredPermission = PermissionManager.DEBUG_PERMISSION)
```

Comment:
Is this permission not required after all? Or why was it removed?

Reply:
Code inspection told me that `DEBUG_PERMISSION` is the default and suggested omitting it.